target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -395,7 +393,9 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0:lan:1" "1:lan:2" "2:lan:3" "3:lan:4" "6@eth0"
 		;;
-	netgear,r6120)
+	netgear,r6120|\
+	wevo,11acnas|\
+	wevo,w2914ns-v2)
 		ucidef_add_switch "switch0" \
 			"0:lan:4" "1:lan:3" "2:lan:2" "3:lan:1" "4:wan" "6@eth0"
```

Comment:
Won't this result in a collision for port 4 in luci?

Reply:
Hmm it seemed okay when I tested...

Reply:
Okay, I've also found other similar entries in 02_network.
However, let me just change my question: What is your reason to remove the ":5" for the wan port?

Reply:
AFAIK port indices are only used for sorting ethernet ports in the same role.
https://github.com/openwrt/luci/blob/master/modules/luci-base/htdocs/luci-static/resources/network.js#L541
For this model, there is only one WAN port, and the index number "5" does not affect in any way, for now.
Still it can be leaved if you think we can use port index in some other ways in the future.

Reply:
I see your point. Still, if the ":5" has no negative effect, removing it looks to me like removing information with no reason for it. So I'd keep it.
However, to be honest I'm aware that the subject isn't worth the big discussion we are having here. So I will also be able to sleep at night if you remove it :-)